frontend/src/sass/_animations.scss

```diff
@@ -0,0 +1,8 @@
+@keyframes loader-spin {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
```

Comment:
newline at the end of file

Reply:
Done

Reply:
There you go. ;-)

![image](https://user-images.githubusercontent.com/17498341/36215463-792e5dbc-11ac-11e8-83f1-cd2286cb019b.png)